config/ses.toml

```diff
@@ -3,11 +3,11 @@
 # The target version of SES to install. This must match a key below that
 # contains the repositories and yaml substitutions.
 
-ses_target = "ses7"
+ses_target = "devel"
 
   [ses.ses7]
     [ses.ses7.yaml_substitutions]
-      "suse.com" = "suse.de/suse/containers/ses/7/containers"
+      # "suse.com" = "suse.de/devel/storage/7.0/containers"
```

Comment:
Why commented out?

Reply:
This would be for testing released versions.